rest/config_test.go

```diff
@@ -957,6 +957,7 @@ func TestParseCommandLineWithIllegalOptionBucket(t *testing.T) {
 }
 
 func TestPutInvalidConfig(t *testing.T) {
+	defer base.SetUpTestLogging(base.LevelDebug, base.KeyAll)()
```

Comment:
What's changed to make this required for a test that doesn't otherwise require logging?

Reply:
I thought adding logging would be useful here initially since this test was failing earlier when running with race detecter turned on. But this can be removed.